sources.xenial.list

```diff
@@ -10,5 +10,8 @@ deb-src http://archive.ubuntu.com/ubuntu xenial-security main restricted univers
 deb http://archive.ubuntu.com/ubuntu xenial-backports main restricted universe multiverse
 deb-src http://archive.ubuntu.com/ubuntu xenial-backports main restricted universe multiverse
 
+#This is for 32 bit version updates
+deb http://security.ubuntu.com/ubuntu xenial-security main universe
```

Comment:
I doubt we need this additional source repository, this repo contains security fixes done by Ubuntu maintainers for packages that are found in the `archive.ubuntu.com` repository. Stable versions are found at the later one, and therefore I doubt whether we need it here.

Resource for further reading:
https://askubuntu.com/questions/185317/are-security-ubuntu-com-updates-eventually-merged-into-normal-updates/185444#185444

Reply:
The repo should be ok. 

wget from the repo above is then obsolete.

Reply:
I have added this line if we want to use aptitude to install the package, the archive mirrors for the 32bit versions for the archive mirror are not in sync with the master repository, this is for the chromium repo as mentioned: https://packages.ubuntu.com/xenial/i386/chromium-browser/download
I have tested installing this package on my desktop with the archive mirror, as expected it took a long time to install, maybe to it was the syncing time I will still check this for travis
wget from the repo to get the package will solve all these problems as you can just get the package with the given url